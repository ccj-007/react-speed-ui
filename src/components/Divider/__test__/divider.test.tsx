import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Divider } from '../../index';

describe('test Divider component', () => {
  it('should render', () => {
    const { asFragment } = render(<Divider />);
    expect(asFragment()).toMatchSnapshot();
  });
});
