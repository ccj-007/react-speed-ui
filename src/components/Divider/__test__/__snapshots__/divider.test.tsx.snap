// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Divider component should render 1`] = `
<DocumentFragment>
  <div
    class="divider divider-solid divider-horizontal"
  />
</DocumentFragment>
`;
